javav2/example_code/dynamodbasync/src/main/java/com/example/dynamodbasync/AsyncPagination.java

```diff
@@ -194,33 +189,34 @@ public void onNext(String tableName) {
             }
 
             @Override
-            public void onError(Throwable t) { }
+            public void onError(Throwable t) {
+            }
 
             @Override
-            public void onComplete() { }
-            // snippet-end:[dynamodb.java2.async_pagination.subscriber]
+            public void onComplete() {
+            }
         });
+        // snippet-end:[dynamodb.java2.async_pagination.subscriber]
 
-        // As the above code is non-blocking, make sure your application doesn't end immediately
-        // For this example, I am using Thread.sleep to wait for all pages to get delivered
+        // Since the code above is non-blocking, make sure your application doesn't end immediately.
+        // For this example, I use Thread.sleep to wait for all pages to get delivered.

```

Comment:
```suggestion
        // Because the preceding code is non-blocking, make sure your application doesn't end immediately.
        // For this example, use Thread.sleep to wait for all pages to get delivered.

```